drone/exec/exec.go

```diff
@@ -327,6 +327,7 @@ func exec(c *cli.Context) error {
 			workspacePath = c.String("workspace-path")
 		}
 		dir, _ := filepath.Abs(filepath.Dir(file))
+		volumes = append(volumes, "drone_default:"+workspaceBase)
```

Comment:
If the user customizes the `--prefix` the volume might have a different name. So we should just take the prefix into account here:

```
c.String("prefix") + "_default"
```

but otherwise LGTM 😄